src/indent.cpp

```diff
@@ -2105,7 +2105,7 @@ void indent_text(void)
             }
 
             /* [Unity-only hack] Do nothing. We're using 'define' as CT_PP_IGNORE to avoid reformatting of complex and weird macros. Likewise leave indentation alone because probably also weird. */
```

Comment:
comment is out of date. and also should we just look at using pp_ignore_define_body around this and no longer consider it a "unity hack"?